db/libdwarf

```diff
@@ -0,0 +1,15 @@
+R2PM_BEGIN
+
+R2PM_GIT "git://git.code.sf.net/p/libdwarf/code" libdwarf-code
+R2PM_DESC "install libdwarf"
```

Comment:
drop install from the description, because this pkg can be also uninstalled :P and everybody expects a package to be installable. you may specify the source of this dwarf thing in the description